main.lua

```diff
@@ -1378,8 +1378,13 @@ function loadconfig(nodefaultconfig)
 	else
 		return
 	end
-	
-	s1 = s:split(";")
+
+	if s:find(";\n") then
+		s1 = s:split(";\n")
+	else
+		s1 = s:split(";")
+	end
```

Comment:
actually this might cause issues with text editors that automatically insert newlines at the end of files